packages/insomnia-url/yarn.lock

```diff
@@ -0,0 +1,3642 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I'm assuming this is just an accidental leftover?

Reply:
huh.  yes.  shouldn't be there in the first place.  I'll definitely delete.